Source/Engine/Physics/PhysX/PhysicsBackendPhysX.cpp

```diff
@@ -3083,6 +3083,147 @@ int32 PhysicsBackend::MoveController(void* controller, void* shape, const Vector
 
 #if WITH_VEHICLE
 
+PxVehicleDifferential4WData CreatePxVehicleDifferential4WData(const WheeledVehicle::DifferentialSettings settings)
+{
+    PxVehicleDifferential4WData differential4WData;
+    differential4WData.mType = (PxVehicleDifferential4WData::Enum)settings.Type;
+    differential4WData.mFrontRearSplit = settings.FrontRearSplit;
+    differential4WData.mFrontLeftRightSplit = settings.FrontLeftRightSplit;
+    differential4WData.mRearLeftRightSplit = settings.RearLeftRightSplit;
+    differential4WData.mCentreBias = settings.CentreBias;
+    differential4WData.mFrontBias = settings.FrontBias;
+    differential4WData.mRearBias = settings.RearBias;
+    return differential4WData;
+}
+
+PxVehicleDifferentialNWData CreatePxVehicleDifferentialNWData(const WheeledVehicle::DifferentialSettings settings, const Array<WheeledVehicle::Wheel*, FixedAllocation<PX_MAX_NB_WHEELS>> wheels)
```

Comment:
Pass larger value types such as `DifferentialSettings` via reference - `const DifferentialSettings&`. The same for array to be passed as reference to avoid memory copies.

Reply:
Can verify again ?

Reply:
Missing reference character `&` so the structure will be passed as pointer under the hood without memory copy over the stack.

Reply:
@mafiesto4 yes, i made this fix on last commit, can check ?